packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -347,12 +370,6 @@ export type MutationCheckoutSessionArgs = {
 };
 
 
-export type MutationCreateEventArgs = {
```

Comment:
Had to rename "createEvent" which was a function used mainly for telemetry (might one day be used for user-facing audit logs too). The name was conflicting with the new event object